### PR TITLE
Added Sundarharaincha Municipality Website

### PR DESCRIPTION
**Pull request by gaurab-khanal**
## What kind of change does this PR introduce? 

- [ ] Bug Fix
- [ ] Documentation change
- [x] Added a new website
- [ ] Other(s)

### CODE REVIEW

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]
Can you add some more websites?

**Comment by gaurab-khanal** [comment]
> Can you add some more websites?

Yeah, but I need some time to gather information

**Comment by thenishantsapkota** [comment]
> > Can you add some more websites?
> 
> Yeah, but I need some time to gather information

Please add them in this same pull request

**Comment by gaurab-khanal** [comment]
> > > Can you add some more websites?
> > 
> > 
> > Yeah, but I need some time to gather information
> 
> Please add them in this same pull request i.e. atleast 2 per PR

Ok. Can I add government school/college websites as well?

**Comment by thenishantsapkota** [comment]
> > > > Can you add some more websites?
> > > 
> > > 
> > > Yeah, but I need some time to gather information
> > 
> > 
> > Please add them in this same pull request i.e. atleast 2 per PR
> 
> Ok. Can I add government school/college websites as well?

You can only add websites ending in `.gov.np`

**Review by thenishantsapkota — APPROVED** [review]
LGTM!

**Comment by thenishantsapkota** [comment]
Please resolve all the merge conflicts @gaurab-khanal!

**Review by susanhsrestha — APPROVED** [review]

File: data/websites.json [hunk 1]
```diff
@@ -770,6 +770,11 @@
       "url": "http://nationaldata.gov.np/",
       "description": "National Data Portal Nepal"
     },
+    {
+      "name": "Nepal Government",
```
Comment (susanhsrestha) [inline data/websites.json:774]: Nepal Government' official website is already on the data list.

Comment (susanhsrestha) [inline data/websites.json:774]: Nepal Government' official website is already on the data list.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

File: data/websites.json (hunk 1 above)
Comment (thenishantsapkota) [inline data/websites.json:774]: Can you please remove this entry?
The website is already on the list.

**Review by thenishantsapkota — CHANGES_REQUESTED** [review]

File: data/websites.json (hunk 1 above)
Comment (gaurab-khanal) [reply data/websites.json:774]: > Can you please remove this entry? The website is already on the list.
I re-checked the existence of that website(https://nepal.gov.np/) but I didn't find another one. Can you please show me where is the other one?

**Comment by thenishantsapkota** [comment]
Closing this due to inactivity!

**Pull request closed without merging**